packages/net/dns/resolve.pony

```diff
@@ -1,3 +1,6 @@
+use "net"
+use "net/ip"
+
 type DNSLookupAuth is (AmbientAuth | NetAuth | DNSAuth)
 
 primitive DNS
```

Comment:
I think this file should be named `dns.pony` (even though it is already in a `dns` subdir), because the principal type declared in the file is `DNS`.